src/gwkokab/inference/analyticallikelihood.py

```
@@ -2,16 +2,17 @@
 # SPDX-License-Identifier: Apache-2.0
 
 
-from typing import Callable, Dict, Optional, Tuple, TypeAlias
+from typing import Callable, Dict, Tuple, TypeAlias
```

Ruff (0.14.2)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)